lib/membrane_http_adaptive_stream/manifest/changeset.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Membrane.HTTPAdaptiveStream.Manifest.Changeset do
+  @moduledoc """
+  Structure representing changes that has been applied to the track. What element has been added
+  and what elements are to be removed.
+  """
+  defmodule Segment do
```

Comment:
Could that be under `Manifest` module or is it necessary to be under `Changeset`?